android/build.gradle

```diff
@@ -10,18 +10,19 @@ def computeVersionName() {
 buildscript {
     repositories {
         jcenter()
+        google()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.+'
+        classpath 'com.android.tools.build:gradle:3.1.4'
```

Comment:
Should this file have the `buildscript` entry? I thought the application controls the build tools. I believe that since this is a library, it should not need to include the `repositories` or `buildscript`. Those entries are in the application's `/android/build.gradle` file and are applied to all child projects.

Reply:
I only keep it because it was originally in the code from master branch, as I think removing it might cause side-effects. It doesn't harm anyway.